cloudsmith/assets/dashboards/cloudsmith_overview.json

```diff
@@ -0,0 +1 @@
nse_format":"timeseries","queries":[{"query":"avg:cloudsmith.bandwidth_used{*}","data_source":"metrics","name":"query1"}],"display_type":"line"}],"yaxis":{"include_zero":true,"label":"","scale":"linear","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":0,"width":4,"height":2}},{"id":5336196919922036,"definition":{"title":"Bandwidth Usage Status","title_size":"16","title_align":"left","type":"check_status","check":"cloudsmith.bandwidth","grouping":"check","group":"host:docker-desktop","group_by":[],"tags":["*"]},"layout":{"x":4,"y":0,"width":2,"height":2}},{"id":1390985517334205,"definition":{"title":"Storage Used","title_size":"16","title_align":"left","show_legend":false,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"response_format":"timeseries","queries":[{"query":"avg:cloudsmith.storage_used{*}","data_source":"metrics","name":"query1"}],"display_type":"line"}],"yaxis":{"include_zero":true,"label":"","scale":"linear","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":2,"width":4,"height":2}},{"id":8357871334831905,"definition":{"title":"Storage Usage Status","title_size":"16","title_align":"left","type":"check_status","check":"cloudsmith.storage","grouping":"check","group":"host:docker-desktop","group_by":[],"tags":["*"]},"layout":{"x":4,"y":2,"width":2,"height":2}},{"id":1745585157577347,"definition":{"title":"cloudsmith.token_download_total","title_size":"16","title_align":"left","show_legend":false,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"response_format":"timeseries","queries":[{"query":"avg:cloudsmith.token_download_total{*}","data_source":"metrics","name":"query1"}],"display_type":"line"}],"yaxis":{"include_zero":true,"label":"","scale":"linear","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":4,"width":4,"height":2}},{"id":5290157003767938,"definition":{"title":"cloudsmith.token_count","title_size":"16","title_align":"left","time":{},"type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"data_source":"metrics","name":"query1","query":"avg:cloudsmith.token_count{*}","aggregator":"avg"}],"response_format":"scalar"}],"autoscale":true,"precision":2},"layout":{"x":4,"y":4,"width":3,"height":2}},{"id":5049902402844905,"definition":{"title":"cloudsmith.token_bandwidth_total","title_size":"16","title_align":"left","show_legend":false,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"response_format":"timeseries","queries":[{"query":"avg:cloudsmith.token_bandwidth_total{*}","data_source":"metrics","name":"query1"}],"display_type":"line"}],"yaxis":{"include_zero":true,"label":"","scale":"linear","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":6,"width":4,"height":2}}],"template_variables":[],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"tft-ejb-txv"}
```

Comment:
Hi Ciara, could you review our Dashboarding best practices guide here: https://datadoghq.dev/integrations-core/guidelines/dashboards/#best-practices

It would be great to have the CloudSmith logo anchored to the top as well as some logical grouping of the widgets + explanations.

Reply:
Thanks, @scseanchow- I"ll get a Cloudsmith logo and I'll review the best practices for Dashboard to make it more useful. 

I probably need to group together the Tokens, Bandwidth and Storage items separately